packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -356,6 +366,13 @@ export class EventSourceMapping extends cdk.Resource implements IEventSourceMapp
 
     let consumerGroupConfig = props.kafkaConsumerGroupId ? { consumerGroupId: props.kafkaConsumerGroupId } : undefined;
 
+    let filters: Array<{[key: string]: string}> = [];
```

Comment:
Even though `props.filters` is deprecated, it still needs to work. Changing the previous behavior is also a breaking change. I think what you need is to say is

```ts
const filters: Array<{[key: string]: string}> = props.filters;
if (...) {
  filters.push()
}
```

Reply:
You're right @kaizencc 

I will make some changes to be sure that change doesn't break the previous implementation

Thank's for you reply